$NetBSD: patch-do,v 1.3 2019/09/02 18:20:50 schmonz Exp $

Avoid spurious output.
Run self-test verbosely, to try to get more info from failing bulk builds.

--- do.orig	2019-01-02 19:49:33.000000000 +0000
+++ do
@@ -11,6 +11,8 @@
 # redo itself does.
 #
 
+unset CDPATH
+
 die() {
 	echo "$0:" "$@" >&2
 	exit 42
@@ -50,7 +52,7 @@ fi
 
 build() {
 	./minimal/do -c bin/all || die "failed to compile redo."
-	bin/redo bin/all || die "redo failed self test."
+	bin/redo -vv -x bin/all || die "redo failed self test."
 }
 
 clean() {
